packages/x-components/src/x-modules/search/components/index.ts

```diff
@@ -7,9 +7,5 @@ export { default as Promoted } from './promoted.vue';
 export { default as PromotedsList } from './promoteds-list.vue';
 export { default as Redirection } from './redirection.vue';
 export { default as ResultsList } from './results-list.vue';
-export { default as SortDropdown } from './sort-dropdown.vue';
```

Comment:
We should avoid using the barrel files internally but I think we must keep the exports for the people using the package from the outside. Am I right @diegopf? 🤔

Reply:
Restored